cmd/renew_certificate.go

```diff
@@ -37,8 +37,9 @@ var (
 
 func RenewCertificateCmd() *cobra.Command {
 	command := &cobra.Command{
-		Use:   "renew-certificate",
-		Short: "Rotates Dapr root certificate of your kubernetes cluster",
+		Use:     "renew-certificate",
+		Aliases: []string{"renew-cert", "rc"},
```

Comment:
Since it's suffixed with `mtls`, i.e. `dapr mtls rc`, I guess it should be contextual. I am ok with keeping or removing it.

Reply:
I added it because I have personally felt pain in writing complete `renew-certificate` always while testing. I am in favour of keeping it. 
or we can put "rnc" instead of rc?

Reply:
let's go with rc itself ... 
Can you add usage examples for this with the alias also. 
@msfussell Any issue with having an alias like rc so that we can call `dapr mtls rc` instead of `dapr mtls renew-certificate` ?